src/modules/0.0.12/aws_rds/entity/rds.ts

```diff
@@ -32,6 +32,7 @@ export class RDS {
   })
   allocatedStorage: number;
 
+  // TODO: Tie this to the AvailabilityZone entity in the future
```

Comment:
we should probably create issues for this TODOs?

Reply:
Sounds good.